### ISSUE TITLE
Why is OBELICS generally better than MINT-1T (HTML)?

### ISSUE THREAD

**Issue by lijinginfo** [mlfoundations/MINT-1T#4]
Why is OBELICS generally better than MINT-1T (HTML)? Is the main advantage of MINT-1T over OBELICS primarily related to PDFs? 
<img width="626" alt="Snipaste_2024-06-24_19-42-27" src="https://github.com/mlfoundations/MINT-1T/assets/56179942/c304c6e2-5219-49e1-9cd9-73b59793cd2e">
<img width="507" alt="Snipaste_2024-06-24_19-46-45" src="https://github.com/mlfoundations/MINT-1T/assets/56179942/f871adaf-cf7a-4716-8560-98fcf8a8e435">

**Comment by anas-awadalla** [comment]
Hi @lijinginfo! Thanks for your interest and sorry it took so long to get to this. I was waiting for us to release our latest report so I can share more solid evidence as to why MINT-1T (HTML) is closer to OBELICS.

One thing I will say is that the results in v1 paper were flawed in a few ways: 1) we did not ablate demonstration separators for the in-context learning prompts. 2) We did not include vqav2 evals in the "a" plot.

We reran results for MINT-1T (HTML) and OBELICS using both an XGen-MM and Idefics2 architectures and find that performance is very close. Here are some numbers:

![html-perf-barplots](https://github.com/user-attachments/assets/c2efca91-27f3-4e71-b4a0-817018dcd9b1)

We also reran experiments for the HTML + PDF from MINT-1T and now see much bigger performance gap!
![mint_shot_scaling_twitter](https://github.com/user-attachments/assets/b3529429-5e84-4052-ab5c-7793671143a7)

Thanks!
